spec/requests/api/v1.0/endpoints_spec.rb

```diff
@@ -91,7 +91,7 @@ def instance_path(id)
 
     context "get" do
       it "success: with a valid id" do
-        instance = Endpoint.create!(payload.merge(:tenant => tenant))
+        instance = create(:endpoint, tenant: tenant)
```

Comment:
Is the separate tenant argument needed in these specs?

Reply:
I kept the explicit, externally defined tenant (not defined by Factorybot) when the tenant needed to be accessed in an `expect`. In this case:
```
expect(response).to have_attributes(
          :status => 200,
          :parsed_body => a_hash_including("id" => instance.id.to_s, "tenant" => tenant.external_tenant)
        )
```